apps/ftu/index.html

```diff
@@ -107,6 +108,11 @@
   <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css" />
   <script defer src="/shared/elements/gaia-header/dist/gaia-header.js"></script>
 
+  <!-- Late Customization -->
+  <script defer src="/shared/js/icons_helper.js"></script>
+  <script defer src="./js/late_customization.js"></script>
+  <link rel="stylesheet" href="./style/late_customization.css">
```

Comment:
tiny and picky nit:
you don't need specify the '.' on paths like `./shared ...` or `./js`